tedpermission/src/main/java/com/gun0912/tedpermission/PermissionBuilder.java

```diff
@@ -153,9 +187,27 @@ public T setDeniedCloseButtonText(@StringRes int stringRes) {
         return setDeniedCloseButtonText(getText(stringRes));
     }
 
-    public T setScreenOrientation(int requestedOrientation) {
+    public T setScreenOrientation(@ScreenOrientation int requestedOrientation) {
         this.requestedOrientation = requestedOrientation;
         return (T) this;
     }
 
+    @IntDef({
+            SCREEN_ORIENTATION_UNSPECIFIED,
+            SCREEN_ORIENTATION_LANDSCAPE,
+            SCREEN_ORIENTATION_PORTRAIT,
+            SCREEN_ORIENTATION_USER,
+            SCREEN_ORIENTATION_BEHIND,
+            SCREEN_ORIENTATION_SENSOR,
+            SCREEN_ORIENTATION_NOSENSOR,
+            SCREEN_ORIENTATION_SENSOR_LANDSCAPE,
+            SCREEN_ORIENTATION_SENSOR_PORTRAIT,
+            SCREEN_ORIENTATION_REVERSE_LANDSCAPE,
+            SCREEN_ORIENTATION_REVERSE_PORTRAIT,
+            SCREEN_ORIENTATION_FULL_SENSOR
```

Comment:
it looks need to  add followings.

- [ ] SCREEN_ORIENTATION_FULL_USER
- [ ] SCREEN_ORIENTATION_LOCKED
- [ ] SCREEN_ORIENTATION_USER_LANDSCAPE
- [ ] SCREEN_ORIENTATION_USER_PORTRAIT

Reply:
These ones are not supported in APIs <= 17 so I have to put a `@SuppressWarning` annotation